Gemfile

```diff
@@ -3,7 +3,13 @@ source 'https://rubygems.org'
 gem 'rake'
 gem 'minitest'
 gem 'minitest-reporters'
-gem 'http'
+gem 'minitest-focus'
+gem 'capybara'
+gem 'capybara_minitest_spec'
+# gem 'selenium-webdriver'
```

Comment:
are these two commented-out bits to be added later / in category with TODO comments in other files? may warrant an explanatory comment?